index.html

```diff
@@ -259,6 +259,33 @@ <h3>let's connect</h3>
             </li>
         </ul>
       </section>
+
+      <section id="contact">
+        <div class="contact-bg-form">
+          <div class="contact-form">
+            <h1>Contact me</h1>
+            <p>
+              If you have an application you are 
+              interested in developing, a feature that
+               you need built or a project that needs
+                coding. I’d love to help with it!
+            </p>
+            <form action="/" method="post">
+              <div class="inputs">
```

Comment:
- [ ] Great job on implementing the contact form as specified in the project requirement! 👏 However, there are minor issues 🐞 that we need to address and fix before moving on to the next milestone. To do that kindly consider the below suggestions: 
   - [ ] As specified in the project requirements, consider creating an account at [Formspree](https://formspree.io/html/) where you're going to get a unique link that you can include in your form `action` to collect user input as without an action you can't do that.